### DISCUSSION TITLE
Text Input Modals?

### DISCUSSION THREAD

**Discussion by TheHusseler** [discord-jda/JDA#2088] [Questions and Help] [answered]
Has this been updated to support the new modal components in any way? Or is this in the works?

For reference:
https://discord.com/developers/docs/interactions/message-components#text-inputs

**Comment by RedcodesDev** [comment, marked as answer]
This is currently in the works in #2024. Modals are a big new feature to develop, and it requires some time.

**Comment by R-K-25** [comment]
It is now fully implemented in alpha-11
